frontend/src/bundles/common/components/slider/slider.tsx

```diff
@@ -0,0 +1,64 @@
+import { Box, Slider } from '@mui/material';
+import { styled } from '@mui/material/styles';
+import { useCallback } from 'react';
+
+import { getValidClassNames } from '../../helpers/helpers.js';
+import { defaultExperienceMarks } from './default-values.js';
+import classes from './style.module.css';
+import { defaultSliderStyle } from './styles.js';
+
+type Mark = {
+    label?: string;
+    value: number;
+};
```

Comment:
Name it more general like Option or something like that, because in this case this name bind us with some context.